src/clients/QBittorrent.ts

```diff
@@ -271,30 +292,28 @@ export default class QBittorrent implements TorrentClient {
 			if (await this.isInfoHashInClient(newTorrent.infoHash)) {
 				return InjectionResult.ALREADY_EXISTS;
 			}
+
```

Comment:
are the rest of the changes in this file necessary? I'm not really finding the ternaries more readable here